ZeroWasteFriendly/Scenes/EcoMap/EcoMapCreator.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  EcoMapCreator.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 20/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+import CoreData
```

Comment:
In most of apps I try to make a kind of a Adapter so I do nto have to import CoreData everywhere, but only in one file, or one group fo files in a single folder.

Make a coredatahelper or something